src/dpo/src/legalize_shift.h

```diff
@@ -81,8 +81,6 @@ class ShiftLegalizer
   void merge(Clump* r);
   bool violated(Clump* r, Clump*& l, int& dist);
 
-  ShiftLegalizerParams& params_;
```

Comment:
Should ShiftLegalizerParams be removed?  Similarly with other param classes

Reply:
Here are the warns I got for these params class members:

`OpenROAD/src/dpo/src/legalize_shift.h:84:25: warning: private field 'params_' is not used [-Wunused-private-field]
  ShiftLegalizerParams& params_;`

`OpenROAD/src/dpo/src/detailed_reorder.h:70:18: warning: private field 'rt_' is not used [-Wunused-private-field]
  RoutingParams* rt_;`

`OpenROAD/src/dpo/src/detailed_displacement.h:86:18: warning: private field 'rt_' is not used [-Wunused-private-field]
  RoutingParams* rt_;`

`OpenROAD/src/dpo/src/detailed_hpwl.h:97:18: warning: private field 'rt_' is not used [-Wunused-private-field]
  RoutingParams* rt_;`

`OpenROAD/src/dpo/src/detailed_abu.h:110:18: warning: private field 'rt_' is not used [-Wunused-private-field]
  RoutingParams* rt_;`

```
OpenROAD/src/dpo/src/detailed_orient.h:90:18: warning: private field 'rt_' is not used [-Wunused-private-field]
  RoutingParams* rt_;
```

It seems to me that the RoutingParams were copied and pasted from other classes.

Reply:
Should ShiftLegalizerParams be removed?

Reply:
Yes, it should, and I removed it in my last commit. I didn't get the question before, sorry.